openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/7-csi-functional/HNOV-Backup-Restore-Cstor/backup-restore-cstor-csi

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+
+pod() {
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd e2e-konvoy && bash openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/7-csi-functional/HNOV-Backup-Restore-Cstor/backup-restore-cstor-csi node '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$CI_COMMIT_SHA'"' '"'$RELEASE_TAG'"' '"'$VELERO_VERSION'"' '"'$VELERO_PLUGIN_VERSION'"' '"'$SDK_TOKEN'"'
+}
+
+node() {
+source ~/.profile
+gittoken=$(echo "$github_token")
+
+job_id=$(echo $1)
+pipeline_id=$(echo $2)
+commit_id=$(echo $3)
+releaseTag=$(echo $4)
+velero_version=$(echo $5)
+velero_plugin_version=$(echo $6)
+sdk_token=$(echo $7)
+case_id=HNOV
+
+time="date"
+current_time=$(eval $time)
+
+present_dir=$(pwd)
+echo $present_dir
+
+bash openebs-konvoy-e2e/utils/e2e-cr jobname:backup-restore-cstor-csi jobphase:Waiting
+bash openebs-konvoy-e2e/utils/e2e-cr jobname:backup-restore-cstor-csi jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" openebs_version:"$releaseTag"
```

Comment:
@shashank855  where is the Waiting `e2e-cr` for backup restor using diss profile and backup restore diff ns

Reply:
Look at script remote-restore-diif-ns line-no: 27 & backup-restore-diff-profile line-no: 27

Reply:
For pooling logic we no need to create e2e-cr in the very first dependent job? @shashank855

Reply:
@nsathyaseelan If you look at:
 https://github.com/mayadata-io/e2e-openshift/blob/a696ca677162f9943fc419bb48bbe0de5d690360/Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/3PZ0-Backup-Restore-Cstor/backup-restore-cstor#L27
It works fine in e2e-openshift pipeline.

Reply:
IMO it will not break.